dib/src/pages/MainPage.css

```diff
@@ -7,4 +7,44 @@
 .rest-list-header {
 font-weight: 300;
 margin: 1rem;
+}
+
+.search-container {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.button-base {
+  padding: 10px 25px;
+  border-radius: 12px;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;  
+  transition: background-color 0.3s ease, border-color 0.3s ease, color 0.3s ease;
+  border: 2px solid rgb(175, 171, 171);
+}
+
+.button-base:hover {
+  background-color: #ccc;
+  border-color: #999;  
+  color: #111;  
```

Comment:
for colors lets try to use the variables created in the index.css, to change them quickly globally. you can add also new variables. And if oyu want to change a color just for fun, go ahead! ; )